### DISCUSSION TITLE
Add ability to compare changes from Source and Destination during deployment

### DISCUSSION THREAD

**Discussion by craftanalytix** [TabularEditor/TabularEditor3#1299] [Ideas]
### Is your feature request related to a problem? Please describe

There is no visibility of what has been changed. making sure the changes made is accurate and no accidental changes happen that is not intended to

### Describe the solution you'd like

Visibility of the changes from Source to Destination during deployment and the ability to choose what changes to deploy.

### Goals

1. Understanding what is the changes
2. Avoid  accidental changes or unnecessary changes
3. Deploy only what is ready to move to next phase while developing other changes
4.

**Comment by mlonsk** [comment]
Hi @RoadDeal 
Thank you for the suggestion, it is definitely something we are looking at, but I cannot give any more specifics at this moment.
One workaround right now is to use C# to open ALM toolkit inside TE3 directly. There is an example of how to do that here: https://github.com/TabularEditor/TabularEditor3/discussions/1088#discussioncomment-7264131
